components/compositing/compositor.rs

```diff
@@ -1483,7 +1483,7 @@ impl<Window: WindowMethods> IOCompositor<Window> {
             _ => initialize_png(&*self.gl, width, height)
         };
 
-        profile(ProfilerCategory::Compositing, None, self.time_profiler_chan.clone(), || {
+        profile(ProfilerCategory::Compositing, None, &self.time_profiler_chan.clone(), || {
```

Comment:
Hmm ok then, I was considering changing the closures, or working with the move keyword, but the first option would be less readable, and the second one would be neither correct nor performant. I'll revert my changes, I wish I could find an elegant way to refactor it but I can't find any :/